docs/sql_service.md

```diff
@@ -0,0 +1,116 @@
+# SQL Service
+
+The SQL Service, developed by Semiotic Labs in collaboration with TheGuild
```

Comment:
This sentence can be confusing. I recommend splitting the two ideas up. Something like:

"... provides a secure SQL interface for querying a subgraph's entities. To deploy this with minimal changes to the existing indexer stack, we do this by wrapping the SQL query that a consumer sends in a GraphQL query. For example, given a consumer SQL query [], we create a corresponding GraphQL query []."

Reply:
Also, it's "The Guild," not "TheGuild"